static/js/stream_list.js

```diff
@@ -461,9 +461,10 @@ exports.initialize = function () {
         if (overlays.is_active()) {
             ui_util.change_tab_to('#home');
         }
-        var stream = $(e.target).parents('li').attr('data-name');
+        var stream_id = $(e.target).parents('li').attr('data-stream-id');
+        var stream = stream_data.get_sub_by_id(stream_id);
```

Comment:
Yeah, I'm all in favor of stream -> stream_name, since we are mostly good about that.  But I think we need to think carefully about migrating away from `sub`.  We definitely want to do it, just not now, and I'm not sure `stream` is exactly ideal.